Rubberduck.Parsing/Inspections/Resources/InspectionsUI.resx

```diff
@@ -803,4 +803,13 @@ If the parameter can be null, ignore this inspection result; passing a null valu
     <value>Stop keyword detected</value>
     <comment>{0} Property name</comment>
   </data>
+  <data name="ShadowedDeclarationInspectionMeta" xml:space="preserve">
+    <value>Two declarations have the same name, and both are in scope. This means that only one of them will be available to use.</value>
```

Comment:
Same name, but not *in same scope* - wording seems off. How about:

> Two declarations are in-scope and have the same identifier name. This means that only one of them will be available to use.

Reply:
There is no **same** word, just *in scope*. But I like your wording better anyway, so I'll change it.

Reply:
Oh wow, I'm going blind... sorry about that, I totally read it wrong!